examples/mobile/UIComponents/components/etc/content/spin.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<meta content="width=device-width, user-scalable=no" name="viewport" />
+	<link href="../../../lib/tau/mobile/theme/default/tau.css" rel="stylesheet" />
+	<link href="../../../css/style.css" rel="stylesheet" />
+	<script data-build-remove="false" src="../../../lib/tau/mobile/js/tau.js">
+	</script>
+</head>
+
+<body>
+	<div class="ui-page" id="spin-demo">
+		<header>
+			<h1>
+				Spin
+			</h1>
+		</header>
+		<div class="ui-content">
+			<style>
+				.default-spin {
+					height: 150px;
+					font-size: 50px;
+					width: 200px;
+				}
+				.default-spin .ui-spin-item-selected {
+					font-size: 50px;
+				}
+				.ui-slider {
+					width: 100%;
+				}
+				li {
+					padding: 10px;
+				}
+			</style>
+			<div class="ui-content">
+				<ul class="ui-listview ui-content-area">
+					<li>
+						<div class="ui-spin default-spin"
+							min="0"
+							max="15"
```

Comment:
if you have range from 0..15 an set number to 15 (2 digts) and change digits number up/down behaviour of sample is strange
shouldn't we start in exemplary app with spin range 1..9 and digits range 1..5 ?

Reply:
> if you have range from 0..15 an set number to 15 (2 digts) and change digits number up/down behaviour of sample is strange
> shouldn't we start in exemplary app with spin range 1..9 and digits range 1..5 ?

I don't see nothing strange. This is only test for live update of widget option.